File: temp/fix_all_quotes.py
import os

def fix_file(content):
    # Replace Chinese curly quotes with corner brackets inside JS strings
    # U+201C = "  U+201D = "
    content = content.replace('“', '「')  # " -> 「
    content = content.replace('”', '」')  # " -> 」
    return content

fixed = 0
for f in os.listdir('.'):
    if f.startswith('slide-') and f.endswith('.js'):
        with open(f, 'rb') as fh:
            raw = fh.read()
        try:
            content = raw.decode('utf-8')
        except:
            continue
        original = content
        content = fix_file(content)
        if content != original:
            with open(f, 'w', encoding='utf-8') as fh:
                fh.write(content)
            fixed += 1
            print(f'Fixed: {f}')
print(f'\nTotal: {fixed} files fixed')

File: temp/fix_all_quotes.js
const fs = require('fs');
const path = require('path');

const dir = "D:/新课开发/工作手册/高潜员工加速成长路径设计/完整课程包/02-授课PPT/slides";

const files = [
  'slide-25.js','slide-26.js','slide-28.js','slide-29.js','slide-38.js',
  'slide-44.js','slide-56.js','slide-57.js','slide-88.js','slide-91.js',
  'slide-93.js','slide-106.js','slide-107.js'
];

files.forEach(f => {
  const fp = path.join(dir, f);
  if (!fs.existsSync(fp)) return;
  
  let content = fs.readFileSync(fp, 'utf8');
  
  // Replace escaped quotes with regular quotes
  content = content.replace(/\'/g, "'");
  content = content.replace(/\\"/g, '"');
  
  // Replace Chinese quotes with single quotes (safe for JS strings)
  content = content.replace(/"/g, "'").replace(/"/g, "'");
  
  // But we need to handle cases where Chinese quotes are inside strings
  // If we changed outer quotes of a string that contains Chinese quotes, revert those
  // This is tricky - let's try a different approach: escape Chinese quotes within strings
  
  fs.writeFileSync(fp, content);
  console.log('Fixed: ' + f);
});
